src/strategies/hats-protocol-single-vote-per-org/index.ts

```diff
@@ -0,0 +1,134 @@
+import { subgraphRequest } from '../../utils';
+import { utils } from 'ethers';
+import { StaticJsonRpcProvider } from '@ethersproject/providers';
+import { multicall } from '../../utils';
```

Comment:
```suggestion
import { multicall, Multicaller } from '../../utils';
```

Reply:
I don't think that I want to import the Multicaller from the utils, as the implementation found in this file itself is a modified version of it.